lib/src/zap/zap.h

```diff
@@ -554,6 +559,37 @@ zap_err_t zap_close(zap_ep_t ep);
  */
 zap_err_t zap_send(zap_ep_t ep, void *buf, size_t sz);
 
+/**
+ * \brief Send data to peer using map.
+ *
+ * This is a more efficient send interface comparing to \c zap_send(). The data
+ * to be sent is described by \c map, \c buf and \c len. The data will not be
+ * copied out to the internal zap buffer (unlike \c zap_send()) and the
+ * application should not modify the data in the specified range (buf[0..len-1])
+ * before the send operation is completed. The completion of the send operation
+ * results in \c ZAP_EVENT_SEND_COMPLETE event with the specified \c context
+ * delivered via the callback function specified in \c zap_new() or \c
+ * zap_accept(). The completion status could be success or failed.
+ *
+ * If the function call returns \c ZAP_ERR_OK, it is guaranteed that \c
+ * ZAP_EVENT_SEND_COMPLETE corresponding to the call will be delivered. On the
```

Comment:
Nix "on the other hand"

Reply:
I'm confused. Do you mean to merge the first sentence and the second sentence (starting with 'On the other hand') together?